internal/k8s/helpers.go

```diff
@@ -41,6 +40,17 @@ func NewClusterRoleBinding(name, namespace, serviceAccount string) *rbacv1.Clust
 	}
 }
 
+func NewIngressClass(name string) *v1.IngressClass {
+	return &v1.IngressClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: name,
+		},
+		Spec: v1.IngressClassSpec{
+			Controller: defaults.IngressControllerName,
+		},
+	}
+}
```

Comment:
Let's change the ingress controller name to be passed as an argument, akin to other functions in this package:

```suggestion
func NewIngressClass(name, controllerName string) *v1.IngressClass {
	return &v1.IngressClass{
		ObjectMeta: metav1.ObjectMeta{
			Name: name,
		},
		Spec: v1.IngressClassSpec{
			Controller: controllerName,
		},
	}
}
```